l2-frontend/src/fields/ConfigureAnesthesiaField.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div style="margin-top: 10px" v-model="tb_data" :readonly="disabled">
+    <table class="table table-bordered">
+      <thead>
+      <tr>
+        <th class="first-column">Наименование</th>
+        <th class="second-column">Категория</th>
+        <th class="third-column">Показать по умолчанию</th>
+        <th></th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr class="tr-body" v-for="(val, index) in tb_data">
+        <td class="first-column"><input class="no-outline" type="text" style="width:100%" v-model="val['title']"
+                                        placeholder="Введите наименование"></td>
+        <td class="second-column">
+          <select v-model="val['type']">
+            <option disabled value="">Выберите один из вариантов</option>
+            <option>Показатели человека</option>
+            <option>Сильнодействующие</option>
+            <option>Наркотические</option>
+          </select>
+        </td>
+        <td class="third-column" align="center">
+          <input type="checkbox" v-model="val['default']">
+        </td>
+        <td>
+          <button class="btn btn-default btn-primary-nb" v-if="tb_data.length > 0" v-on:click="delete_row(index)"
```

Comment:
Снова v-on:, есть же @
```suggestion
          <button class="btn btn-default btn-primary-nb" v-if="tb_data.length > 0" @click="delete_row(index)"
```

Reply:
А зачем тут `v-if="tb_data.length > 0`?
Ведь это всё итерируется в рамках как раз tb_data, условие будет всегда истинным

Reply:
раньше проверка была >1: что если есть тип поля то одно значение будет точно! потом отказался уберу условие